### DISCUSSION TITLE
Will Studio features come to GrapesJS editor

### DISCUSSION THREAD

**Discussion by bernesto** [GrapesJS/grapesjs#5699] [General]
The new Studio product looks great. I was wondering if there is a plan to bring the editor features of Studio like the draggable padding and margin, updated styles, and locked layers to Editor as well? Those are very well executed.

Thank you!

**Comment by artf** [comment]
I'd say not at the current stage.

**Reply by bernesto** [reply]
Okay. I'd be glad to help bring some of them to it.
